src/main/java/org/apache/mesos/state/AbstractStateStore.java

```diff
@@ -0,0 +1,42 @@
+package org.apache.mesos.state;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.*;
+
+/**
+ * Abstract StateStore implementation.
+ */
+public abstract class AbstractStateStore implements StateStore {
+    private static final Logger log = LoggerFactory.getLogger(SchedulerState.class);
+
+    public void storePropertyAsObj(final String key, final Object obj) throws StateStoreException {
```

Comment:
These could both be static methods of StateStore and avoid the definition of an abstract class, of which there are none (to my knowledge) in dcos-commons.